diff --git a/test/test.cabal b/test/test.cabal
index 2516098..a6ad185 100644
--- a/test/test.cabal
+++ b/test/test.cabal
@@ -1,5 +1,3 @@
--- Comment: This is the Cabal specification version used for the file.
--- Reserved Keyword: 'cabal-version'
 cabal-version: 3.0
 
 -- Reserved Keyword: 'name'
@@ -15,7 +13,7 @@ version: 0.1.0.0
 synopsis: Minimal Haskell application example.
 
 -- Reserved Keyword: 'description'
-description: This package demonstrates the minimal required fields 
+description: This package demonstrates the minimal required fields
              to create a simple executable in Haskell using Cabal.
 
 -- Reserved Keyword: 'category'
